homeassistant/util/color.py

```diff
@@ -214,6 +221,11 @@ def color_RGB_to_xy(iR: int, iG: int, iB: int) -> Tuple[float, float, int]:
     return round(x, 3), round(y, 3), brightness
 
 
+def color_xy_to_RGB(vX: float, vY: float) -> Tuple[int, int, int]:
```

Comment:
I don't like this. It implies that such a conversion is possible but really it is just `color_xy_brightness_to_RGB` with a default value for brightness.

Having callers use `color_xy_brightness_to_RGB` forces them to think about the brightness to use.

If we don't want `255` sprinkled about, maybe an actual default parameter could be used?

Reply:
Per my other comment, I don't think platforms should be using color attributes to track brightness. We previously forced them to care about it and we got all sorts of differing behaviors. We could rename it to normalized RGB, but that might be a little verbose.